Add master not discovered error for searchability (#6536)
elastic/docs-content: troubleshoot/elasticsearch/discovery-troubleshooting.md

When a node wins the master election, it logs a message containing `elected-as-master` and all nodes log a message containing `master node changed` identifying the new elected master node.

If there is no elected master node and no node can win an election, all nodes repeatedly log messages about the problem using a [logger](/deploy-manage/monitor/logging-configuration.md) called `org.elasticsearch.cluster.coordination.ClusterFormationFailureHelper`. By default, this happens every 10 seconds. 

> During this time, {{es}} returns `MasterNotDiscoveredException`. Its API reports:

Reviewer: ```suggestion
During this time, {{es}} returns `MasterNotDiscoveredException`. The following error displays:
```
